SPECS-SIGNED/kernel-mshv-signed/kernel-mshv-signed.spec

```diff
@@ -3,11 +3,14 @@
 %ifarch x86_64
 %global buildarch x86_64
 %endif
+%ifarch aarch64
+%global buildarch aarch64
+%endif
 %define uname_r %{version}-%{release}
 Summary:        Signed MSHV-enabled Linux Kernel for %{buildarch} systems
 Name:           kernel-mshv-signed-%{buildarch}
 Version:        6.6.100.mshv1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
validation for changes to kernel-mshv-signed: https://dev.azure.com/mariner-org/mariner/Mariner%20Core%20Devs/_build/results?buildId=965275&view=logs&j=0ad494dd-6a2a-56e3-e8a5-1c5709d0a5c3&t=086a2050-0b6b-5764-3257-83b5bac4a292&l=1614